.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,60 @@
+## Summary of Changes
+
+Please write a sentence or two summarizing the proposed change.
+
+Addresses [ORCA-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/ORCA-XXX)
+
+## Changes
+
+* Detailed list or prose of changes
+* ...
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
```

Comment:
```suggestion
Please describe the tests (unit, integration, ad-hoc, or otherwise) that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
```